src/status_im/ui/screens/chat/views.cljs

```diff
@@ -284,77 +311,65 @@
                                       :space-keeper       space-keeper
                                       :chat-id            chat-id
                                       :show-input?        show-input?}
-       :render-fn                    render-fn
+       :render-fn                    render-fn-class
        :on-viewable-items-changed    on-viewable-items-changed
-       ;;TODO this is not really working in pair with inserting new messages because we stop inserting new messages
-       ;;if they outside the viewarea, but we load more here because end is reached,so its slowdown UI because we
-       ;;load and render 20 messages more, but we can't prevent this , because otherwise :on-end-reached will work wrong
-       :on-end-reached               (fn []
-                                       (if @state/scrolling
-                                         (re-frame/dispatch [:chat.ui/load-more-messages chat-id])
-                                         (utils/set-timeout #(re-frame/dispatch [:chat.ui/load-more-messages chat-id])
-                                                            (if platform/low-device? 500 200))))
-
-       :on-scroll-to-index-failed    #()                    ;;don't remove this
+       :on-end-reached               list-on-end-reached
+       :on-scroll-to-index-failed    identity              ;;don't remove this
```

Comment:
Can you add some more context about the `identity`?

Reply:
I just changed anonymous empty function on identity